backend/dal/Services/INotificationQueueService.cs

```diff
@@ -23,7 +23,7 @@ public interface INotificationQueueService : IService
         IEnumerable<NotificationQueue> GenerateNotifications(Project project, int projectStatusNotificationId);
         IEnumerable<NotificationQueue> GenerateNotifications(Project project, ProjectStatusNotification options, DateTime? sendOn = null);
         NotificationQueue GenerateNotification(Project project, ProjectStatusNotification options, Agency agency, DateTime? sendOn = null);
-        NotificationQueue GenerateNotification<T>(string to, NotificationTemplate template, T model, DateTime? sendOn = null);
+        NotificationQueue GenerateNotification<T>(string to, string cc, NotificationTemplate template, T model, DateTime? sendOn = null);
```

Comment:
Just curious will the cc email get sent to the creator/generator of the notification in the DEV & TEST environments?

Reply:
It follows the same rules everything else does.  in DEV and TEST it will CC whomever is testing.   This way at least the tester can confirm the CC value was set.